src/Runner.Sdk/RunnerWebProxy.cs

```diff
@@ -74,8 +74,8 @@ public RunnerWebProxy()
                 _httpProxyAddress = proxyHttpUri.AbsoluteUri;
```

Comment:
can you delete `_httpProxyAddress` since it's no longer used?

Reply:
That still gets used via the `HttpProxyAddress` property in a couple places. Would it make more sense to set `_httpProxyAddress = httpProxyAddress;` instead of setting just the environment variables to it?

Reply:
feel like `_httpProxyAddress = proxyHttpUri.OriginalString;` is better, i pushed it to your branch. 😄

Reply:
oh yeah that's much better. I must've missed that when I was looking at the Uri class